hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/extensions/SpecEntityExtension.java

```diff
@@ -125,7 +125,8 @@ public void beforeAll(@NonNull final ExtensionContext context) throws Exception
 
     private SpecContract contractFrom(@NonNull final Contract annotation) {
         final var name = annotation.name().isBlank() ? annotation.contract() : annotation.name();
-        return new SpecContract(name, annotation.contract(), annotation.creationGas());
+        final var specContract = new SpecContract(name, annotation.contract(), annotation.creationGas());
```

Comment:
Not a big deal but why this change?  It seems equivalent.

Reply:
Its absolutely equivalent, seems like I've missed to revert this change.

 Thank you, I removed it.